src/release/program-github-promote-release.ts

```diff
@@ -28,7 +28,7 @@ const promoteReleases = async ({ deliverinoPrivateKey, owner, repo, platform }:
   const preReleases = releases.filter(release => release.prerelease)
   // For integreat we always want platform android to be the latest release, so a link to the latest github release will go to the apk
   // For entitlementcard we always want platform all to be the lastest release
-  const platformsFlaggedLatest = ['android', 'all']
+  const platformsFlaggedLatest = ['android', 'native', 'all']
```

Comment:
❓ Do we still need android separately, then?

Reply:
I think it still makes sense as the idea is to tag android builds as latest to quickly find the apk (even though its not used atm).